### DISCUSSION TITLE
Word 模板制作探讨

### DISCUSSION THREAD

**Discussion by sikouhjw** [whutug/whu-thesis#98] [General]
闲得无聊想做个 Word 模板（如果 whutug 愿意接收的话）。

不知道需求大不大，应该挺大的？用 LaTeX 的总是少数。

成品参考 gdutthesis-Word[^1]，我看了下规范好像没什么地方难实现的。

[^1]: https://github.com/sikouhjw/gdutthesis-Word/

**Comment by tanukihee** [comment]
readme 里面有规范，但脑壳痛的是各种不按规范来的各院土规定，以及外院基本完全不兼容的另一套）

**Reply by sikouhjw** [reply]
在样式那里设计好，各院的规定应该可以比较简单的修改（存疑），或者可以写到 issue 里让他们对着改

**Comment by xkwxdyy** [comment]
麻烦的是如 @tanukihee 所说，比如外语系的那个英文摘要可能英语系和法语系的整个排版设计都不同，感觉外语系之外的应该不会差太多（吧
